### PR TITLE
Count active browser sessions per user

### PR DESCRIPTION
**Pull request by Tymek**
## About the changes
Show info on how many devices a user is logged in to an admin.

### CODE REVIEW

File: src/lib/types/experimental.ts [hunk 1]
```diff
@@ -283,6 +284,10 @@ const flags: IFlags = {
         process.env.UNLEASH_EXPERIMENTAL_FLAG_OVERVIEW_REDESIGN,
         false,
     ),
+    showUserDeviceCount: parseEnvVarBoolean(
```
Comment (kwasniew) [inline src/lib/types/experimental.ts:287]: seems to be inconsistent. We have user session count not device count in the code

File: src/lib/openapi/spec/user-schema.ts [hunk 2]
```diff
@@ -99,6 +99,12 @@ export const userSchema = {
             nullable: true,
             example: '01HTMEXAMPLESCIMID7SWWGHN6',
         },
+        activeSessions: {
+            description: 'The number of active sessions for this user',
```
Comment (kwasniew) [inline src/lib/openapi/spec/user-schema.ts:103]: We can explain that any browser session counts as active session

Comment (Tymek) [inline src/lib/openapi/spec/user-schema.ts:103]:
```suggestion
            description: 'Count of active browser sessions for this user',
```

File: src/lib/types/stores/session-store.ts [hunk 3]
```diff
@@ -12,4 +12,5 @@ export interface ISessionStore extends Store<ISession, string> {
     getSessionsForUser(userId: number): Promise<ISession[]>;
     deleteSessionsForUser(userId: number): Promise<void>;
     insertSession(data: Omit<ISession, 'createdAt'>): Promise<ISession>;
+    getUserSessionsCount(): Promise<{ userId: number; count: number }[]>;
```
[the remark this answers is not in the document]
Comment (Tymek) [reply src/lib/types/stores/session-store.ts]: `getSessionCount` is OK

**Pull request merged**